pkg/cloud/amazon/eksctl/eksctl_cli.go

```diff
@@ -19,7 +19,7 @@ func NewEksctlClient() eksctlClient {
 
 // DeleteCluster performs an eksctl cluster deletion process
 func (eksctlClient) DeleteCluster(cluster *cluster.Cluster) error {
-	cmd := exec.Command("eksctl", "delete", "cluster", "--name", cluster.Name) //nolint:gosec
+	cmd := exec.Command("eksctl", "delete", "cluster", "--cluster", cluster.Name) //nolint:gosec
```

Comment:
I am not so sure about this. I did an upgrade to the latest eksctl version:
```
$ eksctl version
[ℹ]  version.Info{BuiltAt:"", GitCommit:"", GitTag:"0.11.1"}
```

and it still seems to be _name_:

```
eksctl delete cluster -h
Delete a cluster

Usage: eksctl delete cluster [flags]

General flags:
  -n, --name string          EKS cluster name
  -r, --region string        AWS region
  -w, --wait                 wait for deletion of all resources before exiting
  -f, --config-file string   load configuration from a file (or stdin if set to '-')
      --timeout duration     maximum waiting time for any long-running operation (default 25m0s)

AWS client flags:
  -p, --profile string        AWS credentials profile to use (overrides the AWS_PROFILE environment variable)
      --cfn-role-arn string   IAM role used by CloudFormation to call AWS API on your behalf

Common flags:
  -C, --color string   toggle colorized logs (valid options: true, false, fabulous) (default "true")
  -h, --help           help for this command
  -v, --verbose int    set log level, use 0 to silence, 4 for debugging and 5 for debugging with AWS debug logging (default 3)

Use 'eksctl delete cluster [command] --help' for more information about a command.
```

Reply:
My bad, I think I made a bad assumption in thinking that it would be have been changed across the board when I saw:

```
Flag --name has been deprecated, use --cluster
[ℹ]  eksctl version 0.11.1
```

The change looks to only be with `eksctl utils associate-iam-oidc-provider` and not the other two commands - I'll update my PR, apologies! :)

Reply:
I think the `update-kubeconfig` command is changed to:

```
Usage: eksctl utils write-kubeconfig [flags]

General flags:
  -c, --cluster string     EKS cluster name
  -r, --region string      AWS region
      --timeout duration   maximum waiting time for any long-running operation (default 25m0s)
```